### ISSUE TITLE
No GPG signature file for ISO embedded in split zips

### ISSUE THREAD

**Issue by lsteeger** [manjaro/release-review#15]
Is this even possible with split zips?

**Comment by boredland** [comment]
isn't that what we're doing?
![image](https://user-images.githubusercontent.com/4662748/128758268-c7773ccb-f1d0-4c00-b7ab-347886effbdd.png)

**Comment by lsteeger** [comment]
> isn't that what we're doing?

GPG verification via gpg command requires a .sig file for the --verify option

Reference: https://wiki.manjaro.org/index.php/How-to_verify_GPG_key_of_official_.ISO_images

There are no .sig files present here or inside the resulting combined zip.

**Comment by boredland** [comment]
you're absolutely right, misread your message somehow. I think its just missing the configuration line - the iso action supports this: https://github.com/manjaro/manjaro-iso-action/tree/ci-test-ignore-me-18#gpg-signing
